qa/L0_parameters/parameters_test.py

```diff
@@ -54,13 +54,14 @@ async def asyncSetUp(self):
         self.async_grpc = asyncgrpcclient.InferenceServerClient(url="localhost:8001")
 
         self.parameter_list = []
-        self.parameter_list.append({"key1": "value1", "key2": "value2"})
-        self.parameter_list.append({"key1": 1, "key2": 2})
-        self.parameter_list.append({"key1": 123.123, "key2": 321.321})
-        self.parameter_list.append({"key1": True, "key2": "value2"})
-        self.parameter_list.append({"triton_": True, "key2": "value2"})
-
-        if TEST_HEADER == "1":
+        if TEST_HEADER in ["0", "1"]:
```

Comment:
I don't understand why `parameter_list` is constructed under condition, why it is not needed in case "2"?